symfony/client/src/core/components/dialogs/ConfirmationDialog.vue

```diff
@@ -0,0 +1,131 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <teleport to="#app">
+    <oxd-dialog
+      v-if="show"
+      :style="{maxWidth: '450px'}"
```

Comment:
better to move this inline style to a class, inside this SFC

Reply:
Moved to a class but had to remove `scoped` on the style tag for the style to apply. Shall I move all the styles to a separate .scss since both `DeleteConfirmationDialog` and `ConfirmationDialog` use them?